`plat/xen`: Build error on Xen ARM64: undefined reference to `prepare_console`

### Describe the bug

Building for Xen on ARM64 results in error. This bug was introduced by #1464.

### Steps to reproduce

Build C Hello in the [`catalog-core` repository[(https://github.com/unikraft/catalog-core). Follow the steps:

1. Clone the repository:

   ```console
   git clone https://github.com/unikraft/catalog-core
   ```

1. Enter the `c-hello/` directory:

   ```console
   cd catalog-core/c-hello/
   ```
   
1. Switch to the `scripts` branch:

   ```console
   git checkout scripts
   ```

1. Build C Hello for Xen/ARM64:

   ```console
   ./build.xen.arm64
   ```

   The build error occurs.

### Expected behavior

Build doesn't fail.

### Which architectures were you using or does this bug affect?

arm64

### Which platforms were you using or does this bug affect?

xen

### Relevant log output

```shell
/usr/lib/gcc-cross/aarch64-linux-gnu/13/../../../../aarch64-linux-gnu/bin/ld: /home/razvan/orgs/unikraft/tmp/test-nginx-core/catalog-core/c-hello/out/c-hello_xen-arm64.elf.o: in function `_libxenplat_armentry':
/home/razvan/orgs/unikraft/tmp/test-nginx-core/catalog-core/repos/unikraft/plat/xen/arm/setup64.c:361: undefined reference to `prepare_console'
/home/razvan/orgs/unikraft/tmp/test-nginx-core/catalog-core/repos/unikraft/plat/xen/arm/setup64.c:361:(.text+0x2210): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `prepare_console'
/usr/lib/gcc-cross/aarch64-linux-gnu/13/../../../../aarch64-linux-gnu/bin/ld: /home/razvan/orgs/unikraft/tmp/test-nginx-core/catalog-core/repos/unikraft/plat/xen/arm/setup64.c:363: undefined reference to `init_console'
/home/razvan/orgs/unikraft/tmp/test-nginx-core/catalog-core/repos/unikraft/plat/xen/arm/setup64.c:363:(.text+0x2214): relocation truncated to fit: R_AARCH64_CALL26 against undefined symbol `init_console'
collect2: error: ld returned 1 exit status
```

--- a/plat/xen/arm/setup64.c
+++ b/plat/xen/arm/setup64.c
@@ -339,9 +339,6 @@ void _libxenplat_armentry(void *dtb_pointer, paddr_t physical_offset)
 	/* Set up events. */
 	init_events();
 
-	/* Do early init */
-	uk_boot_early_init(bi);
-
 	/* Initialize logical boot CPU */
 	r = lcpu_init(lcpu_get_bsp());
 	if (unlikely(r))
@@ -358,9 +355,8 @@ void _libxenplat_armentry(void *dtb_pointer, paddr_t physical_offset)
 	get_console();
 	get_xenbus();
 
-	prepare_console();
-	/* Init console */
-	init_console();
+	/* Do early init */
+	uk_boot_early_init(bi);
 
 	uk_boot_entry();
 }

